rotkehlchen/api/v1/resources.py

```diff
@@ -1576,3 +1581,17 @@ class ERC20TokenInfo(BaseResource):
     @use_kwargs(get_schema, location='json_and_query')
     def get(self, address: ChecksumEthAddress, async_query: bool) -> Response:
         return self.rest_api.get_token_information(address, async_query)
+
+
+class BinanceAvailableMarkets(BaseResource):
+    def get(self) -> Response:
+        return self.rest_api.get_all_binance_pairs()
+
+
+class BinanceUserMarkets(BaseResource):
+
+    get_schema = BinanceMarketsUserSchema()
+
+    @use_kwargs(get_schema, location='json_and_query_and_view_args')
+    def get(self, name: str, location: Location) -> Response:
```

Comment:
Why do you need the name in this schema? Binance will have same pairs for all api keys. Binance US also.

Reply:
Because the setting is set for pairs `(name, location)` and I find it nice since you can have multiple binance accounts and select different markets